solidity/hardhat.config.ts

```diff
@@ -11,10 +11,15 @@ const config: HardhatUserConfig = {
     // public LUKSO Testnet
     networks: {
       luksoTestnet: {
-        url: "https://rpc.testnet.lukso.network",
+        url: "https://lukso-testnet.rpc.thirdweb.com",
```

Comment:
how is it different to official one?

Reply:
<img width="807" alt="Screenshot 2024-02-15 at 11 29 41 AM" src="https://github.com/yearone-io/universal-grave/assets/128236767/f11b3e27-8e8e-4463-9c0b-feb90990f208">
Lukso themselves recommends using one of the other providers